<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BodyWindow</class>
 <widget class="QWidget" name="BodyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body Motion</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblMusic">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PlayMusic</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMotion">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/conductor.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSongLyric">
       <property name="font">
        <font>
         <family>細明體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Song Lyric</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
